### PR TITLE
add dic doc

### PR DESCRIPTION
**Pull request by vmidyllic**

### CODE REVIEW

File: verifiable/did_doc.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	Context []string  `json:"context"`
+	ID      string    `json:"id"`
+	Service []Service `json:"service"`
+}
+
+// Service DID document service field.
+type Service struct {
+	ID              string `json:"id"`
+	Type            string `json:"type"`
+	ServiceEndpoint string `json:"serviceEndpoint"`
```
Comment (ilya-korotya) [inline verifiable/did_doc.go:14]: In most places, we use `snake_case`. https://github.com/iden3/go-schema-processor/blob/main/verifiable/proof.go But here we use `camelCase`. Can we define a common standard for JSON tags? 
P.S. Here we use `snake_case` and `camelCase` together.

File: verifiable/did_doc.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package verifiable
+
+// DIDDocument defines current supported did doc model.
+type DIDDocument struct {
+	Context []string  `json:"context"`
```
Comment (ilya-korotya) [inline verifiable/did_doc.go:5]: In general we use `@context` for context field

**Review by ilya-korotya — APPROVED** [review]

File: verifiable/did_doc.go (hunk 2 above)
Comment (vmidyllic) [reply verifiable/did_doc.go:5]: that's right.

Comment (vmidyllic) [reply verifiable/did_doc.go]: done

**Pull request merged**